lib/chromic_pdf.ex

```diff
@@ -278,6 +278,23 @@ defmodule ChromicPDF do
         [chrome_args: "--font-render-hinting=none"]
       end
 
+  In some cases, chromic_pdf's default args may conflict with the ones you would like to add.
+
+  E.g. `--disable-gpu`
+
+  The problem is that you can't simply override them like so:
+
+  E.g. in default args within chromic_pdf's code, `--disable-gpu` and then
+  passing `chrome_args: "--enable-gpu"` won't work.
+
+  The `:conflicting_args` option allows targeted removing of problematic default
+  args so that they don't disrupt your chrome configuration.
+
+      defp chromic_pdf_opts do
+        [conflicting_args: ["--headless", "--disable-gpu"],
```

Comment:
How about, instead of introducing a new option that doesn't relate to chrome anymore, we extend the values the `chrome_args` option can take? E.g. we could make it a `String.t() | [append: String.t(), remove: String.t()]` or so?

Reply:
> How about, instead of introducing a new option that doesn't relate to chrome anymore, we extend the values the chrome_args option can take? E.g. we could make it a String.t() | [append: String.t(), remove: String.t()] or so?

Like it! 

Made the suggested changes and pushed with the slight variation that `:remove` takes a list instead of string.